doc/classes/Animation.xml

```diff
@@ -5,15 +5,26 @@
 	</brief_description>
 	<description>
 		An Animation resource contains data used to animate everything in the engine. Animations are divided into tracks, and each track must be linked to a node. The state of that node can be changed through time, by adding timed keys (events) to the track.
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		# This creates an animation that makes the node "Enemy" move to the right by
 		# 100 pixels in 0.5 seconds.
 		var animation = Animation.new()
 		var track_index = animation.add_track(Animation.TYPE_VALUE)
 		animation.track_set_path(track_index, "Enemy:position:x")
 		animation.track_insert_key(track_index, 0.0, 0)
 		animation.track_insert_key(track_index, 0.5, 100)
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		// This creates an animation that makes the node "Enemy" move to the right by
+		// 100 pixels in 0.5 seconds.
+		var animation = new Animation();
+		int trackIndex = animation.AddTrack(Animation.TYPE_VALUE);
```

Comment:
```suggestion
		int trackIndex = animation.AddTrack(Animation.TrackType.Value);
```
Enums are PascalCase and are in the format `EnumName.Value` instead of `ENUMNAME_VALUE`.

Reply:
fixed